src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -60,7 +60,7 @@ public class RedisHubLifetimeManager<THub> : HubLifetimeManager<THub>, IDisposab
             else
             {
                 // TODO: We could support reconnecting, like old SignalR does.
-                throw new InvalidOperationException("Connection to redis failed.");
+                //throw new InvalidOperationException("Connection to redis failed.");
```

Comment:
what is this.

Reply:
Yes what is this

Reply:
This will go away when #1037 is merged which I'm waiting for before this PR